velox/exec/tests/PrestoQueryRunnerTest.cpp

```diff
@@ -95,4 +99,44 @@ TEST_F(PrestoQueryRunnerTest, DISABLED_fuzzer) {
   velox::exec::test::assertEqualResults(
       prestoResults, plan->outputType(), {veloxResults});
 }
+
+TEST_F(PrestoQueryRunnerTest, sortedOrder) {
```

Comment:
Should we disable this test by default?

Reply:
I kept it enabled since I was able to run this without a local presto instance listening on 8080. It doesnt seem like we eagerly establish a connection when we initialize PQR, should that change this will fail. I can still DISABLE it if you insist though.